dotnet/src/Connectors/Connectors.AI.OpenAI/TextToImage/OpenAITextToImage.cs

```diff
@@ -53,7 +53,7 @@ public sealed class OpenAIImageGeneration : IImageGeneration
         this._authorizationHeaderValue = $"Bearer {apiKey}";
         this._organizationHeaderValue = organization;
 
-        this._core = new(httpClient, loggerFactory?.CreateLogger(typeof(OpenAIImageGeneration)));
+        this._core = new(httpClient, loggerFactory?.CreateLogger(typeof(OpenAITextToImage)));
```

Comment:
nit. This will resolve the problem without having to touch this code again for future renamings.

```suggestion
        this._core = new(httpClient, loggerFactory?.CreateLogger(this.GetType()));
```